egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -951,12 +951,18 @@ if [ ${stage} -le 5 ] && [ ${stop_stage} -ge 5 ] && ! [[ " ${skip_stages} " =~ [
             log "Error: not supported SOT training for whisper token_list"
             exit 2
         fi
+
+        _opts=""
+        if [ "${token_type}" = "whisper_multilingual" ]; then
+            _opts+=" --language ${lang}"
+        fi
+
         # The first symbol in token_list must be "<blank>" and the last must be also sos/eos:
         # 0 is reserved for CTC-blank for ASR and also used as ignore-index in the other task
         echo ${token_list}
         ${python} -m espnet2.bin.whisper_export_vocabulary  \
             --whisper_model "${token_type}" \
-            --output "${token_list}"
+            --output "${token_list}" ${_opts}
```

Comment:
I think it would be useful if the script can exit when ${lang} is not recognized. Is this already satisfied? Or just by adding `|| exit 1` after this line. I'm not sure if the python scripts satisfies the condition of retuning non-zero status.

Reply:
Unfortunately, the shell script will directly terminate when `whisper_export_vocabulary` raises the "ValueError: language unsupported for Whisper model". I am not sure how to handle this situation.
Using `|| exit 1` or not does not affect the shell script.
![image](https://github.com/espnet/espnet/assets/23135280/49b8a671-7026-4d63-9448-be1ffc925e65)